### PR TITLE
added product page with filter

### PR DESCRIPTION
**Pull request by Tejs1**

### CODE REVIEW

File: pages/products.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Tarandeep-s1ngh) [inline pages/products.html:7]: You should update the title of this page. For instance, "Products | Expanse"

File: pages/products.html [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  </head>
+  <body>
+    <nav class="km-nav-bar" id="nav">
+      <div class="km-nav-logo"><h1>Expanse</h1></div>
+      <div class="km-right-nav">
+        <div class="km-nav-searchbar">
+          <input type="search" placeholder="Search..." class="km-input" />
+          <button>
+            <a href="" class="cta-btn">
+              <div></div>
```
Comment (Tarandeep-s1ngh) [inline pages/products.html:22]: Any specific reason for using this extra empty div here?

File: styles/shop.css [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+}
+#store-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  justify-content: left;
+}
+#store-wrapper .ship-item {
+  height: 290px;
+  width: 820px;
```
Comment (simran61) [inline styles/shop.css:26]: Try using relative units instead of px

**Review by simran61 — APPROVED** [review]

**Review by ayush65 — APPROVED** [review]

File: styles/auth.css [hunk 4]
```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css?family=Muli&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0%;
+  padding: 0%;
+}
+body {
```
Comment (Tarandeep-s1ngh) [inline styles/auth.css:8]: It is recommended to write this in reset.css in order to avoid repetition in every CSS file.

File: styles/shop.css [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+  position: relative;
+  margin: 40px 0px 120px 0;
+  border-bottom: 1px solid #1b4268;
+  border-top: 1px solid #1b4268;
```
Comment (Tarandeep-s1ngh) [inline styles/shop.css:30]: Suggestion: Variables can be used for colors

File: styles/shop.css [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+  width: 901px;
+  height: 136px;
+  background: url("./img/ships/filet_bg.png") center center;
+}
+.center .filet .name {
+  position: absolute;
+  left: 65px;
+  top: 50%;
+  margin-top: -17px;
+  line-height: 34px;
```
Comment (Tarandeep-s1ngh) [inline styles/shop.css:77]: It's better to use relative units like "rem" rather than absolute units

**Review by aniketxparihar — APPROVED** [review]
Looks Good. Just a suggestion try to use rem's or em's instead of pixels.

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: pages/products.html (hunk 1 above)
Comment (Tejs1) [reply pages/products.html:7]: Did it Thanks Taran

**Pull request merged**